apiserver/facades/agent/uniter/uniter_test.go

```diff
@@ -3143,10 +3143,10 @@ func checkUnorderedActionIdsEqual(c *gc.C, ids []string, results params.StringsW
 func (s *uniterSuite) TestStorageAttachments(c *gc.C) {
 	// We need to set up a unit that has storage metadata defined.
 	ch := s.AddTestingCharm(c, "storage-block")
-	sCons := map[string]state.StorageConstraints{
+	sDirectives := map[string]state.StorageConstraints{
```

Comment:
Let's leave this as it was. No need for churn given we're not renaming `state.StorageConstraints`